File: Service_eHospital_DongNai_A/Implement/DM_KeHoachDeAn_1816Service.cs
namespace Service_eHospital_DongNai_A.Implement
{
    public class DM_KeHoachDeAn_1816Service : BaseService<DM_KeHoachDeAn_1816, IDM_KeHoachDeAn_1816Repository>
    , IDM_KeHoachDeAn_1816Service
    {
    private readonly IDM_KeHoachDeAn_1816Repository _DM_KeHoachDeAn_1816Repository;
    public DM_KeHoachDeAn_1816Service(IDM_KeHoachDeAn_1816Repository DM_KeHoachDeAn_1816Repository) : base(DM_KeHoachDeAn_1816Repository)
    {
    _DM_KeHoachDeAn_1816Repository = DM_KeHoachDeAn_1816Repository;
    }
    }
    }

File: Report/src/app/shared/eHospital_DongNai_A/DM_KeHoachDeAn_1816.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { DM_KeHoachDeAn_1816 } from './DM_KeHoachDeAn_1816.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class DM_KeHoachDeAn_1816Service extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'KeHoach_Id', 'MaCongViec', 'TenCongViec', 'NoiDung', 'DiaDiem', 'HinhThuc', 'NgayBatDau', 'NgayKetThuc', 'KetQua', 'NgayTao', 'NguoiTao_Id', 'NgayCapNhat', 'NguoiCapNhat_Id'];

    List: DM_KeHoachDeAn_1816[] | undefined;
    ListFilter: DM_KeHoachDeAn_1816[] | undefined;
    FormData!: DM_KeHoachDeAn_1816;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "DM_KeHoachDeAn_1816";
    }
}

